lib/bolt/outputter/human.rb

```diff
@@ -96,12 +96,15 @@ def print_result(result)
           end
         end
 
-        if result.message
-          @stream.puts(remove_trail(indent(2, result.message)))
-        end
+        # Only print results if there's something other than empty string and hash
+        if !result.message? && result.generic_value.empty?
```

Comment:
I actually think we just want to check if `result.value.empty?`, because we don't want to print this message if there was an `error_hash` _or_ in the `ApplyResult` case above.

Reply:
Actually I guess it's more like `result.value.empty? || (result.value.keys == ['_output'] && !result.message?)` which is a little wonky. That is, if the value is completely empty _or_ if the only value is an empty message.